secure-app-model/keyvault/partnerconsent/src/main/java/com/microsoft/store/samples/partnerconsent/security/KeyVaultProvider.java

```diff
@@ -63,89 +53,24 @@ public String getSecret(String secretName)
      */
     public void setSecret(String secretName, String value)
```

Comment:
Javadoc for this method should say:
Adds a secret  with the specified {@code secretName} and {@code value} to the key vault if it does not exist. If the named secret exists, a new version of the secret is created.